File: cSharp/day30.cs

using System;
public static int[] roll() {
    Random dice = new Random();

    int dice1 = dice.Next(1,7);

    int dice2 = dice.Next(1,7);

    return [dice1, dice2];
}

int[] result = roll();

Console.WriteLine(
    "Dice: " + 
    result[0].ToString() +
    "," +
    result[1].ToString()
);

File: go/day30.go
package main

import (
	"fmt"
	"strconv"
	"math/rand"
	"time"
)

func roll() []int {
	return []int{rand.Intn(6)+1, rand.Intn(6)+1}
}

func main() {

	rand.Seed(time.Now().UnixNano())
	
	result := roll();
	fmt.Print(
		"Dice: " + 
		strconv.Itoa(result[0]) +
		"," +
		strconv.Itoa(result[1]) ,
	);

}